.gitignore

```diff
@@ -320,3 +320,5 @@ tags
 .ionide
 
 # End of https://www.toptal.com/developers/gitignore/api/c,c++,vim,python,platformio,visualstudiocode,buck,scons
+
+dbc/
```

Comment:
put this at the top instead of the bottom pls, with the other things that aren't part of the generated output from gitignore.io